tests/hikari/test_channels.py

```diff
@@ -185,6 +185,16 @@ async def test_TextChannel_history():
     )
 
 
+def test_TextChannel_trigger_typing():
+    app = mock.Mock()
+    app.rest.trigger_typing = mock.Mock()
+    mock_channel = mock.Mock(channels.TextChannel, id=123, app=app)
+
+    channels.TextChannel.trigger_typing(mock_channel)
```

Comment:
mock_class_namespace could have been used here :)

Reply:
Ik, I would have used it, but its to keep consistency with the rest of the tests there (which idk who made them). Im probably going to end up rewriting them to look nicer